### PR TITLE
Spelling and grammar errors corrected

### PR DESCRIPTION
**Pull request by ApheleiaS**
@amye As suggested by community members I used aspell to go through all pages and correct typos and spelling errors in the documentation. 
Done as part of Outreachy first contibution. 
- Akanksha

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -55,13 +55,13 @@ pages:
   - Puppet Gluster: Administrator Guide/Puppet.md
   - RDMA Transport: Administrator Guide/RDMA Transport.md
   - GlusterFS iSCSI: Administrator Guide/GlusterFS iSCSI.md
-  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Intergration.md
+  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Integration.md
```
Comment (humblec) [inline mkdocs.yml]: If we correct above, we need to correct the file name here https://github.com/gluster/glusterdocs/blob/master/Administrator%20Guide/NFS-Ganesha%20GlusterFS%20Intergration.md

**Comment by humblec** [comment]
@ApheleiaS Thanks !!! Please address my review comment.

File: GlusterFS Tools/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-GlusterFS Tools
+Gustavo's Tools
```
Comment (humblec) [inline GlusterFS Tools/README.md]: Whats "Gustavo" ?

File: GlusterFS Tools/glusterfind.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+A tool which helps to get full/incremental list of files/dirs from GlusterFS Volume using Changelog/Find. In Gluster volumes, detecting the modified files is challenging. Readdir on a directory leads to multiple network calls since files in a directory are distributed across nodes.
 
-This tool should be run in one of the node, which will get Volume info and gets the list of nodes and brick paths. For each brick, it spawns the process and runs crawler command in respective node. Crawler will be run in brick FS(xfs, ext4 etc) and not in Gluster Mount. Crawler generates output file with the list of files modified after last run or after the session creation.
+This tool should be run in one of the node, which will get Volume info and gets the list of nodes and brick paths. For each brick, it spawns the process and runs crawler command in respective node. Crawler will be run in brick S(xfs, ext4 etc) and not in Gluster Mount. Crawler generates output file with the list of files modified after last run or after the session creation.
```
Comment (humblec) [inline GlusterFS Tools/glusterfind.md]: I dont understand this ( s/FS/S ? ) change.

File: mkdocs.yml (hunk 1 above)
Comment (ApheleiaS) [inline mkdocs.yml]: In my commit I did rename the file. It shows up in my git commit message but I am not sure why its not showing up in this pull request.

File: mkdocs.yml [hunk 4]
```diff
@@ -55,13 +55,13 @@ pages:
...
   - Linux Kernel Tuning: Administrator Guide/Linux Kernel Tuning.md
   - Troubleshooting: Administrator Guide/Troubleshooting.md
   - Network Configurations Techniques: Administrator Guide/Network Configurations Techniques.md
   - Performance Testing: Administrator Guide/Performance Testing.md
   - Export and Netgroup Authentication: Administrator Guide/Export And Netgroup Authentication.md
-  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Intergration.md
+  - Configuring NFS-Ganesha server: Administrator Guide/NFS-Ganesha GlusterFS Integration.md
```
Comment (ApheleiaS) [inline mkdocs.yml]: @humblec Pls check?

**Pull request merged**